public/templates/teacher_sets/detail.html

```diff
@@ -13,13 +13,13 @@ <h3>{{ts.title}}</h3>
         <div id='total_available_copies' align="center" style="font-weight:bold"> {{ts.available_copies}} of 
           {{ts.total_copies}} Available | {{ts.new_holds_count}} sets pending
         </div><p style="white-space: pre"></p>
-        <div data-ng-show="ts.available_copies > 0 && !active_hold && is_available ">
+        <div data-ng-show="ts.is_copies_available">
```

Comment:
is_copies_available condition is required for to show different text message for  available button and unavailable button.
 eg:

AVAILABLE - ORDER NOW!
Available Teacher Sets will deliver to your school within 2 weeks. For Teacher Sets that are currently in use by other educators, please allow 60 days or more for delivery. If you need materials right away, contact us at help@mylibrarynyc.org.


0 of 2 Available | 7 sets pending
UNAVAILABLE
You have placed a hold on this Teacher Set. If you need multiple copies of this Teacher Set, contact us at help@mylibrarynyc.org.